src/vs/workbench/browser/parts/compositeBar.ts

```diff
@@ -118,11 +113,7 @@ export class CompositeDragAndDrop implements ICompositeDragAndDrop {
 				return dragData.id !== targetCompositeId;
 			}
 
-			// ... to another composite location
-			const draggedViews = this.viewDescriptorService.getViewContainerModel(currentContainer)!.allViewDescriptors;
```

Comment:
@sandy081 and I took a look at this, and the Remote Explorer doesn't work properly with this change.